core/fx/stream.go

```diff
@@ -79,6 +97,11 @@ func Range(source <-chan interface{}) Stream {
 	}
 }
 
+// Concat Returns a concat Stream.
+func Concat(a Stream, others ...Stream) Stream {
+	return a.Concat(others...)
+}
```

Comment:
when we need to concat streams?

Reply:
> when we need to concat streams?

When we want to do different filtering for two different streams, then do the same filtering, and finally output to the same file.